### ISSUE TITLE
Support for thumbnails per object

### ISSUE THREAD

**Issue by shahchintan14** [3MFConsortium/lib3mf#166]
Any plans to explicitly support thumbnails per object? Or can this be supported once custom attributes PR of @qmuntal is merged?

**Comment by martinweismann** [comment]
hi @shahchintan14 ,
looks like object thumbnails are also part of the implementation `void CModelObject::setThumbnail(_In_ std::string sThumbnail)`, but simply not part of the API. Should be included before the release of v2.
